server/chat-plugins/poll.ts

```diff
@@ -306,7 +309,15 @@ export const commands: ChatCommands = {
 			if (new Set(options).size !== options.length) {
 				return this.errorReply("There are duplicate options in the poll.");
 			}
-
+			// if there's a poll in place, queue instead.
```

Comment:
I would probably make it an explicit command instead of doing it automatically, to guard against accidental usage.

Reply:
Removed.